config/routes.rb

```diff
@@ -5,12 +5,13 @@ def draw(routes_name)
     instance_eval(File.read(Rails.root.join("config/routes/#{routes_name}.rb")))
   end
 
-  devise_for :users, path_prefix: "devise", controllers: { registrations: "registrations" }
+  devise_for :users, path_prefix: "devise", controllers: { registrations: "profiles" }
 
   devise_scope :user do
     scope "my" do
-      put "profile/update", to: "registrations#update"
-      put "password/update", to: "passwords#update_password"
+      put "profile", to: "profiles#update"
+      patch "password", to: "passwords#update_password"
```

Comment:
Why are we calling this action `update_password`? Is `update` action of `passwords` controller used for some other purpose?